### ISSUE TITLE
Give OpenCode a Mascot.. ! | Part 1

### ISSUE THREAD

**Issue by Manav1410** [opencodeiiita/Lets-Design-2.0#50]
**OPEN TO ALL
No need to claim this issue
NO SUBMISSION WILL BE ACCEPTED AFTER 12th October 11:59 PM**

</br>
In this task, you have to suggest a mascot for OpenCode and also, state the reason to justify your suggestion. 

**For those of you who don't know, what a mascot is,**

A mascot is a character (animal, object or person) designed to represent a collective, incorporated or not, with common identity and values. This field is between illustration and logo design.

## **How to Create a Mascot:**

1. **Choose the right character**
Humans, animals, or objects – there are several factors you need to consider to choose the right mascot.
2. **Create the personality of your character**
There are several simple questions to follow to build the personality of your character.
3. **Plan Poses, Emotions, Outfits, and Scenes**
Your character working on the computer, or talking on the phone – frame what variations of your mascot you will need.
4. **Design the character**
Now that you're done with the first three steps, it's time to design your mascot and shape your ideas. 

 </br>     

In This task, you have to complete first 3 points. For Designing the Mascot, New Issue will be opened after submission for this task ends.

If you are unclear with how to start, you can check out this article: 
[https://visualcontent.space/how-to-create-a-brand-mascot/](https://visualcontent.space/how-to-create-a-brand-mascot/)

## Submission Guidelines:

Submit your response as a text file.

## MDC (Minimum Design Criteria):

The final decision remains in the hands of mentors and Organizers, based on the effectiveness of the idea, your justification for the idea, and how closely it represents OpenCode.

For any doubts related to the task or the submission process, you can always ping us on discord. We will be happy to help.
